### DISCUSSION TITLE
Type Error

### DISCUSSION THREAD

**Discussion by danoGen** [canopen-python/canopen#458] [Q&A]
In network.py, line 48 is giving me the error 
self.listeners = [MessageListener(self)]

TypeError: Can't instantiate abstract class MessageListener with abstract method stop

Is this due to an upgraded python? Whats going on here?

**Comment by acolomb** [comment]
It's a known issue, see #455. We are going to release a new version very soon that will restore compatibility with the python-can package version 4.4.0. In the meantime, the workaround is to downgrade to python-can 4.3.1.

**Comment by acolomb** [comment]
See also https://github.com/hardbyte/python-can/pull/1795.

The breaking change was already reverted.
